template/blt/project.yml

```diff
@@ -39,15 +44,15 @@ target-hooks:
   # Executed when front end dependencies should be installed.
   frontend-setup:
     # E.g., ${docroot}/sites/all/themes/custom/mytheme.
-    dir: ${docroot}
+    dir: '${docroot}/themes/custom/${project.subtheme.machine_name}'
     # E.g., `npm install` or `bower install`.
-    command: echo 'No frontend-setup configured.'
+    command: 'npm install -g yarn; yarn install;'
   # Executed when front end assets should be generated.
   frontend-build:
     # E.g., ${docroot}/sites/all/themes/custom/mytheme.
-    dir: ${docroot}
+    dir: '${docroot}/themes/custom/${project.subtheme.machine_name}'
     # E.g., `npm run build`.
-    command: echo 'No frontend-build configured.'
+    command: 'gulp'
```

Comment:
This should be executing using an npm script rather than running as a globally available bin.